packages/p2p/src/components/my-ads/create-ad-form.jsx

```diff
@@ -95,6 +95,20 @@ const CreateAdForm = () => {
                 }}
             >
                 {({ errors, handleChange, isSubmitting, isValid, setFieldValue, touched, values }) => {
+                    let invalidAmountError = false;
+
+                    Object.keys(errors).forEach(key => {
+                        // eslint-disable-next-line no-unused-expressions
+                        errors[key] === 'Enter a valid amount' ? (invalidAmountError = true) : undefined;
+                    });
+
+                    if (invalidAmountError) {
+                        Object.keys(errors).forEach(key => {
+                            // eslint-disable-next-line no-unused-expressions
+                            errors[key] === 'Enter a valid amount' ? undefined : (errors[key] = undefined);
+                        });
+                    }
+
```

Comment:
It seems you can do all of these in a simpler form without disabling Eslint
Do you want to make all other `errors[key]` to be `undefined` if one of them is equal to `'Enter a valid amount'`?
By the way, it is not a good pattern to inject assignment inside a turnery operator.

```suggestion
                  let invalidAmountError = Object.values(errors).includes('Enter a valid amount');

                  if (invalidAmountError) {
                      Object.entries(errors).forEach(([key, value]) => {
                          errors[key] = value === 'Enter a valid amount' ? value : undefined;
                      })
                  }
```

Reply:
Of course, it's look obviously simpler and cleaner, thx for the hint! All requested changes are implemented now.